project-api/project-api-test/src/test/java/io/ballerina/projects/test/PackageResolutionTests.java

```diff
@@ -153,14 +153,70 @@ public void testProjectWithInvalidBuildFile() throws IOException {
 
         PackageCompilation compilation = loadProject.currentPackage().getCompilation();
 
+        boolean check = false;
         DependencyGraph<ResolvedPackageDependency> dependencyGraph = compilation.getResolution().dependencyGraph();
         for (ResolvedPackageDependency graphNode : dependencyGraph.getNodes()) {
             Collection<ResolvedPackageDependency> directDeps = dependencyGraph.getDirectDependencies(graphNode);
+            PackageManifest manifest = graphNode.packageInstance().manifest();
```

Comment:
This test case has been written to check the functionality of the build file. We wouldn't expect this to fail due to issues in dependency resolution logic. Shall we write a new test case for dependency resolution assertions?